Makefile

```diff
@@ -47,6 +47,7 @@ unit-tests:
 	@bash ./scripts/unittests.sh
 
 # test runs ALL the test suites.
-test: systemtests unit-tests
+test:
+	@bash ./scripts/tests.sh
```

Comment:
I'm not sure if we need this here. As a developer I would want to return once there is a failure so I can fix them right away rather waiting and checking till the end.